cosmwasm/packages/wasmi-runtime/Makefile

```diff
@@ -108,8 +108,8 @@ ffi-types:
 	RUST_TARGET_PATH=$(Rust_Target_Path) RUSTFLAGS=$(Rust_Flags) xargo build --features "build_headers" --target x86_64-unknown-linux-sgx -Z unstable-options --profile $(BUILD_PROFILE) && \
 	cp $(Rust_Target_Path)/../../../target/headers/enclave-ffi-types.h $(Rust_Target_Path)/../target/headers/
 
-librust_cosmwasm_enclave.signed.so: librust_cosmwasm_enclave.so
-	sgx_sign sign -key Enclave_private.pem -enclave $^ -out $@ -config $(ENCLAVE_CONFIG)
+librust_cosmwasm_enclave.signed.so: librust_cosmwasm_enclave.so $(ENCLAVE_CONFIG)
+	sgx_sign sign -key Enclave_private.pem -enclave $< -out $@ -config $(ENCLAVE_CONFIG)
 
```

Comment:
Note that whether or not we accept this PR, we should include this specific change.

Reply:
So please extract it to a new micro-PR.

Reply:
https://github.com/enigmampc/SecretNetwork/pull/537